vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -170,7 +170,6 @@ def build(self):
                 ActionLoop._create_app_callbacks(),
```

Comment:
The `Output("dashboard-container", "className")` in this file shouldn't be there any more.

Reply:
We currently seem to need it such that the callback is triggered, otherwise the layouts don't match, see: https://github.com/mckinsey/vizro/pull/804#discussion_r1799135660